src/routes/coin-value/coinValue.css

```
@@ -0,0 +1,84 @@
+@import url('https://fonts.googleapis.com/css2?family=Mulish:wght@400;600;700&family=Raleway:wght@300;400;500;600;700;800;900&display=swap');
+:root {
+  --primary-background: #ffffff;
+  --primary-black-text: #230b34;
+  --primary-dark-grey: #3f3f3f;
+  --primary-grey-text: #626262;
+  --primary-green: #21d91d;
+  --primary-border-radius: 40px;
+}
+.dark {
```

It's better to have a blank line between every selector.

```suggestion
}

.dark {
```